src/footer/js/footer.js

```diff
@@ -34,11 +34,11 @@ export default (data) => {
 			$footerBreadcrumb,
 		),
 		$('div', {class: `${prefix}--wrapper`},
-			$('div', {class: `${prefix}-section--1 ${data.hideMenus ? 'hidden' : ''}`},
+			$('div', {class: `${prefix}-section--1 ${data.hideMenus && 'hidden'}`},
 				$footerBrand,
```

Comment:
please revert this to -- $('div', {class: `${prefix}-section--1 ${data.hideMenus ? 'hidden' : ''}`},